app/templates/views/trial-mode.html

```diff
@@ -38,7 +38,7 @@ <h2 class="heading-medium" id="before-you-request-to-go-live">Before you request
     <li>add examples of the messages you want to send</li>
     <li>update your settings so you’re ready to send and receive messages</li>
     <li>accept our data sharing and financial agreement</li>
-    <li>accept our terms of use</li>
+    <li>accept our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.terms') }}">terms of use</li>
```

Comment:
Should there be a closing a tag here? 🤔 👀 
```suggestion
    <li>accept our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.terms') }}">terms of use</a></li>
```

Reply:
sorry, not sure how I missed that when reviewing 🤦‍♀️